disnake/client.py

```diff
@@ -1952,6 +1953,28 @@ async def create_guild(
             data = await self.http.create_guild(name, icon_base64)
         return Guild(data=data, state=self._connection)
 
+    def guild_builder(self, name: str) -> GuildBuilder:
+        """Creates a builder object that can be used to create more complex guilds.
+
+        This is a more comprehensive alternative to :func:`create_guild`.
+        See :class:`.GuildBuilder` for details and examples.
+
+        Bot accounts in 10 or more guilds are not allowed to create guilds.
+
+        .. versionadded:: 2.7
```

Comment:
This is obvious, but well, change this to 2.8 :)

Reply:
Good catch, this PR was quite far behind, hadn't updated it since :sweat_smile:
https://github.com/DisnakeDev/disnake/pull/578/commits/75f9bf54fbad336b3fe4d0ec00f0268c544473ba